lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3524,11 +3421,6 @@ namespace {
       SmallVector<VarDecl *, 4> members;
```

Comment:
In the future we'll be able to refactor and remove _a lot_ of this visitor. But I want to do that in another nfc.

Reply:
I think we will still have to import every member here unless we can make `hasUnreferenceableStorage` lazy, which is unfortunate. But we should be able to remove a lot of the rest of this logic. I'm hoping to move the logic for synthesizing members (such as for unnamed unions) into those member's visitors. That should also help reduce what needs to be done when only importing the class itself.